components/41-organisms/header/header.twig

```diff
@@ -2,10 +2,9 @@
   <div class="content-container">
     {% include '@logo' with {
       site_name: site_name,
-      type: logo_type
+      type: subsite ? 'subsite' : null
```

Comment:
Twig doesn't require you to specify the last value.

```suggestion
      type: subsite ? 'subsite'	
```

Reply:
TIL